src/deepsparse/pipeline.py

```diff
@@ -137,6 +137,12 @@ def __init__(
         self.engine = self._initialize_engine()
 
     def __call__(self, *args, **kwargs) -> BaseModel:
+        if "engine_inputs" in kwargs:
+            raise ValueError(
+                "invalid kwarg engine_inputs. engine inputs determined "
+                f"by {self.__class__.__name__}.parse_inputs"
```

Comment:
Would `__qualname__` be more informative here? Specially for nested classes
```suggestion
                f"by {self.__class__.__qualname__}.parse_inputs"
```